Shared/Extensions/StorageAware+Box.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  StorageAware+Box.swift
```

Comment:
I wanted to just have an in-memory cache for the legacy API objects. That led me down a Swift rabbit hole that created this beast. This is all fine if you just ignore this code and look at the end result of RelistenCache in the ApiCache file